reference/constraints/DivisibleBy.rst

```diff
@@ -3,6 +3,12 @@ DivisibleBy
 
 Validates that a value is divisible by another value, defined in the options.
 
+.. seealso::
+
+    If you need to validate that the number of elements in a collection is
+    divisible by a certain number, use the ``divisibleBy`` option of the
```

Comment:
Maybe "[...] use the `Count` constraint with the `divisibleBy` option"?

Reply:
Much better! Updated. Thanks.